bigbone/src/main/kotlin/social/bigbone/api/method/PreferenceMethods.kt

```diff
@@ -0,0 +1,27 @@
+package social.bigbone.api.method
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.api.entity.Preferences
+import social.bigbone.api.exception.BigBoneRequestException
+
+/**
+ * Allows access to API methods with endpoint equal to an "api/vX/preferences".
+ * @see [Mastodon preferences API methods](https://docs.joinmastodon.org/methods/preferences/)
+ */
+class PreferenceMethods(private val client: MastodonClient) {
```

Comment:
As I only noticed now that I also forgot that in other PRs:

Please add this to `MastodonClient` like

```kotlin
   /**
 * Access API methods under "preferences" endpoint.
 */
@Suppress("unused") // public API
@get:JvmName("preferences")
val preferences: PreferenceMethods by lazy { PreferenceMethods(this) }
```